src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/base/_fonts.scss

```diff
@@ -39,5 +39,5 @@
 }
 
 // Font Awesome config
-$fa-font-path: '../vendor/font-awesome/fonts';
+$fa-font-path: '../fonts';
 
```

Comment:
Die fonts staan nochtans in de map: https://github.com/sumocoders/Framework/tree/master/src/SumoCoders/FrameworkCoreBundle/Resources/assets/fonts, en die worden in web/assets/fonts gezet, en met Gulp worden de tot, wolf, ... files gemaakt.

Dus dan beter de fonts niet mee comitten maar kopiëren uit de `node_modules`?